From 15b7c5ebcbb607cd2edc2119dfefd16b41cddc21 Mon Sep 17 00:00:00 2001
From: babak sarashki <babak.sarashki@windriver.com>
Date: Sun, 3 Nov 2019 14:32:09 -0800
Subject: [PATCH 03/20] Removes unnecessary linking of SQL Libs into slad.

This makes openldap-servers package independent of libodbc (SQL
backend is packaged separately in openldap-servers-sql.)

From STX openldap-sql-linking.patch
---
 build/top.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/top.mk b/build/top.mk
index 3b9e202..4364187 100644
--- a/build/top.mk
+++ b/build/top.mk
@@ -201,7 +201,7 @@ SLAPD_SQL_LDFLAGS = @SLAPD_SQL_LDFLAGS@
 SLAPD_SQL_INCLUDES = @SLAPD_SQL_INCLUDES@
 SLAPD_SQL_LIBS = @SLAPD_SQL_LIBS@
 
-SLAPD_LIBS = @SLAPD_LIBS@ @SLAPD_PERL_LDFLAGS@ @SLAPD_SQL_LDFLAGS@ @SLAPD_SQL_LIBS@ @SLAPD_SLP_LIBS@ @SLAPD_GMP_LIBS@ $(ICU_LIBS)
+SLAPD_LIBS = @SLAPD_LIBS@ @SLAPD_PERL_LDFLAGS@ @SLAPD_SLP_LIBS@ @SLAPD_GMP_LIBS@ $(ICU_LIBS)
 
 # Our Defaults
 CC = $(AC_CC)
-- 
2.17.1

